Sources/RedisMulti+Basic.swift

```diff
@@ -54,6 +54,77 @@ extension RedisMulti {
         queuedCommands.append([RedisString("BITCOUNT"), RedisString(key), RedisString(start), RedisString(end)])
         return self
     }
+    
+    /// Used in BITFIELD
+    public enum BitfieldSubcommand {
+        // (type, offset)
+        case get(String, Int)
+        
+        // (type, offset, value)
+        case set(String, String, Int)
+        
+        // (type, offset, increment)
+        case incrby(String, String, Int)
+        
+        // (wrap/sat/fail)
+        case overflow(BitfieldOverflow)
+        
+        public enum BitfieldOverflow: String {
+            case WRAP, SAT, FAIL
+        }
+    }
+    
+    /// Add a BITFIELD command to the "transaction"
+    ///
+    /// - parameter key: The key of the string to manipulate.
+    /// - parameter subcommands: `BitfieldSubcommand`s to do on the string.
+    ///
+    /// - Returns: The `RedisMulti` object being added to.
+    @discardableResult
+    public func bitfield(key: String, subcommands: BitfieldSubcommand...) -> RedisMulti {
+        return bitfieldArrayOfSubcommands(key: key, subcommands: subcommands)
+    }
+    
+    /// Add a BITFIELD command to the "transaction"
+    ///
+    /// - parameter key: The key of the string to manipulate.
+    /// - parameter subcommands: `BitfieldSubcommand`s to do on the string.
+    ///
+    /// - Returns: The `RedisMulti` object being added to.
+    @discardableResult
+    public func bitfieldArrayOfSubcommands(key: String, subcommands: [BitfieldSubcommand]) -> RedisMulti {
+        var command = ["BITFIELD", key]
+        for subcommand in subcommands {
+            switch(subcommand) {
+            case .get(let type, let offset):
+                command.append("GET")
+                command.append(String(type))
+                command.append(String(offset))
+            case .set(let type, let offset, let value):
+                command.append("SET")
+                command.append(type)
+                command.append(String(offset))
+                command.append(String(value))
+            case .incrby(let type, let offset, let incr):
+                command.append("INCRBY")
+                command.append(type)
+                command.append(String(offset))
+                command.append(String(incr))
+            case .overflow(let overflow):
+                command.append("OVERFLOW")
+                switch(overflow) {
+                case .WRAP:
+                    command.append(BitfieldSubcommand.BitfieldOverflow.WRAP.rawValue)
+                case .SAT:
+                    command.append(BitfieldSubcommand.BitfieldOverflow.SAT.rawValue)
+                case .FAIL:
+                    command.append(BitfieldSubcommand.BitfieldOverflow.FAIL.rawValue)
+                }
```

Comment:
Why do you need the switch here at all? Why not just write:

     command.append(overflow.rawValue)

Isn't overflow of the type `BitfieldSubcommand.BitfieldOverflow`?